std_msgs/Header header

float64 lat
float64 lon
float64 alt
float64 dir
float64 vel
uint8 gear